src/Files.App/Utils/Storage/Operations/FileOperationsHelpers.cs

```diff
@@ -1018,5 +1018,27 @@ protected override void Dispose(bool disposing)
 				}
 			}
 		}
+
+		public static string GetIncrementalName(bool overWriteOnCopy,string? filePathToCheck)
+		{
+			if (filePathToCheck == null)
+				return null;
+			else
+			{
+				if ((!Path.Exists(filePathToCheck)) || overWriteOnCopy)
+					return Path.GetFileName(filePathToCheck);
+				else
+				{
+					int i = 2;
+					string genfilepath = string.Concat([filePathToCheck, " - (", i.ToString(), ")"]);
```

Comment:
`string.Concat([filePathToCheck, " - (", i.ToString(), ")"]);` could be wrapped into a lambda to be called twice to reduce duplication.